torch/_inductor/kernel/flex_attention.py

```diff
@@ -1687,7 +1654,6 @@ def flex_attention(
             mutated_inputs=[
                 logsumexp,
             ],
-            workspace_arg=workspace_arg,
```

Comment:
is this auto-generated now?

Reply:
Triton provides workspace now, so we don’t need to pass it into the tma constructor manually